src/components/listings/MapLocationListing.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import {
+  ImageBackground,
+  ImageSourcePropType,
+  View,
+  Text,
+  Pressable,
+} from 'react-native'
+
+interface MapLocationListingProps {
+  locations: Array<{
```

Comment:
can you pull out this object and make it an interface and use `[]` instead of `Array<>` please?

```
  locations: MapListingLocation[]
```

Reply:
👍